dotcom-rendering/src/web/components/TopicFilterBank.tsx

```diff
@@ -41,34 +41,24 @@ const topicStyles = css`
 	}
 `;
 
-const handleTopicClick = (
-	isActive: boolean,
-	buttonParams: string,
-	id: string,
-) => {
-	const urlParams = new URLSearchParams(window.location.search);
-
-	if (!isActive) {
-		urlParams.set('topics', buttonParams);
-	} else {
-		urlParams.delete('topics');
-	}
-	urlParams.delete('page'); // direct to the first page
-	urlParams.delete('filterKeyEvents');
+const getTopicLink = (isActive: boolean, topics: string, id: Props['id']) => {
+	const urlParams = new URLSearchParams(
+		isActive
+			? {
+					topics,
+			  }
+			: {},
+	);
 
-	window.location.hash = id;
-	window.location.search = urlParams.toString();
+	return `?${urlParams.toString()}#${id}`;
```

Comment:
Is there a reason we don't just hardcode this `id`?

Reply:
Yes, there are two different ones: `key-events-carousel-desktop` & `key-events-carousel-mobile`